clients/command/client.go

```diff
@@ -48,36 +48,25 @@ func NewCommandClient(params types.EndpointParams, m interfaces.Endpointer) Comm
 }
 
 func (cc *commandRestClient) Get(deviceId string, commandId string, ctx context.Context) (string, error) {
-	url, err := cc.urlClient.Prefix()
-	if err != nil {
-		return "", err
-	}
-
-	body, err := clients.GetRequest(url+"/"+deviceId+"/command/"+commandId, ctx)
+	body, err := clients.GetRequest("/"+deviceId+"/command/"+commandId, ctx, cc.urlClient)
 	return string(body), err
 }
 
 func (cc *commandRestClient) Put(deviceId string, commandId string, body string, ctx context.Context) (string, error) {
-	url, err := cc.urlClient.Prefix()
-	if err != nil {
-		return "", err
-	}
-
-	return clients.PutRequest(url+"/"+deviceId+"/command/"+commandId, []byte(body), ctx)
+	return clients.PutRequest("/"+deviceId+"/command/"+commandId, []byte(body), ctx, cc.urlClient)
 }
 
 func (cc *commandRestClient) GetDeviceCommandByNames(
 	deviceName string,
 	commandName string,
 	ctx context.Context) (string, error) {
 
-	url, err := cc.urlClient.Prefix()
+	body, err := clients.GetRequest("/name/"+deviceName+"/command/"+commandName, ctx, cc.urlClient)
 	if err != nil {
 		return "", err
 	}
 
-	body, err := clients.GetRequest(url+"/name/"+deviceName+"/command/"+commandName, ctx)
-	return string(body), err
+	return string(body), nil
```

Comment:
Inconsistent with `Get` implementation above.

Look at creating a `clients.GetRequestReturnString` (not the best name, think about it and come up with something better) that does the cast for you.  This would further simplify code in lots of place from something like what you have above to:

```
return clients.GetRequestReturnString("/name/"+deviceName+"/command/"+commandName, ctx, cc.urlClient)
```